Makefile

```diff
@@ -8,33 +8,35 @@ IMG_TAG ?= latest
 
 EXPORT_RESULT?=false # for CI please set EXPORT_RESULT to true
 
+GOCMD=go
```

Comment:
next thing will be to use devbox

Reply:
> next thing will be to use devbox

Yeah, @faiq is working with Security and Legal so we can get the jetpack devbox install GitHub action allowed.